polars/src/series/mod.rs

```diff
@@ -1145,6 +1179,12 @@ mod test {
         ca.into_series();
     }
 
+    #[test]
+    fn new_series_from_arrow_primitive_array() {
+        let array = UInt64Array::from(vec![1, 2, 3, 4, 5]);
+        let _array_ref: ArrayRef = Arc::new(array);
```

Comment:
unfinished test?

Reply:
Thanks for you quick review.

There is indeed some duplication with `src\frame\ser\mod.rs`. I'll look into that later this week.

I missed a part of the test and plan to shoot in the same commit.